Source/Core/DotSpatial.Data/DotSpatial.Data.csproj

```diff
@@ -1,15 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0-windows</TargetFramework>
-    <Description>GIS Data objects for DotSpatial.</Description>
+      <LangVersion>9.0</LangVersion>
```

Comment:
why not 10.0?

Reply:
Language Version 9 was required, but it could be also 10.
The problem was that for .netstandard, the Language Version was 7.3 by default. And then, there was code that did not compile.